Controllers/OrderSelector.go

```diff
@@ -0,0 +1,48 @@
+package Controllers
+
+import (
+	"fmt"
+	"github.com/Gauravsinghal09/Ecommerce/Models/Order"
+	"sync"
+	"time"
+)
+
+type CustomerMap struct {
+	Map sync.Map
+	//Queue []Order.Orders
+}
+
+var CMap CustomerMap
+
+func OrderQueue(order *Order.Orders) {
+
+	time.Sleep(time.Second * 15)
+	val, ok := CMap.Map.Load(order.CustomerId)
```

Comment:
issue when 3 concurrent request come to create order for some merchant and alll of them with enter if